pkg/k8sinit/testdata/schema/full.yaml

```diff
@@ -47,3 +47,7 @@ containerdRegistryConfigs:
 join:
   url: 10.0.0.10:25000/my-token/hash
   worker: true
+extraCNIEnv:
```

Comment:
So we are only applying IPv4 configs by default?

Reply:
This is not the default configuration that will be applied, this is simply a testing fixture